src/components/experiments/ExperimentTable.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List } from "immutable";
+
+export class ExperimentTable extends React.Component {
+  render() {
+    const formatDate = dateString => {
+      return new Date(dateString)
+        .toUTCString()
+        .replace(/\s*[0-9]{2}:[0-9]{2}:[0-9]{2} GMT$/i, "");
+    };
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Start Date</th>
+            <th>End Date</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          {this.props.filteredExperiments.map(function(item, key) {
```

Comment:
nit: you don't need to make this change but this kind of thing is a little less verbose and imo makes it a little easier to read. the latter point is highly subjective though.

```js
this.props.filteredExperiments.map((item, key) => (
  <tr key={key}>
    <td>{item.get("name")}</td>
    <td>{formatDate(item.get("start_date"))}</td>
    <td>{formatDate(item.get("end_date"))}</td>
    <td>{item.get("status")}</td>
  </tr>
))
```

Reply:
Ah true, thanks for pointing that out!!